.gitignore

```diff
@@ -0,0 +1,9 @@
+*.Rproj
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+.Rbuildignore
```

Comment:
If you add a `.gitignore`, you also need an `.Rbuildignore` to ignore the former during build.